docs/src/pages/docs/crocks/Result.md

```diff
@@ -0,0 +1,1422 @@
+---
+title: "Result"
+description: "Result Crock"
+layout: "guide"
+functions: ["trycatch", "eithertoresult", "firsttoresult", "lasttoresult", "maybetoresult"]
+weight: 130
+---
+
+```haskell
+Result e a = Err e | Ok a
+```
+
+Result is a Sum Type a step above [`Maybe`][maybe]. With a [`Maybe`][maybe] the
+left side represents the lack of a valu, where as with a `Result` the left
+contains
+the error information from an operation. `Result` is well suited for capturing
+disjunction when the cause of the "error" case needs to be communicated. For
+example, when executing a function and you exception is important or useful.
+
+A `Result` represents disjunction by using two constructors, [`Err`](#err) and
+[`Ok`](#ok). An [`Ok`](#ok) instance represents the positive result while
+[`Err`](#err) is considered the negative. With the exception of
+[`coalesce`](#coalesce), [`swap`](#swap) and [`bimap`](#bimap), all `Result`
+returning methods on an instance will be applied to an [`Ok`](#ok) returning the
+result. If an instance is an [`Err`](#err), then all application is skipped and
+another [`Err`](#err) instance is returned with the same containing value.
+
+It is recommended to use the available [`Ok`](#ok) and [`Err`](#err)
+constructors to construct `Result` instances in most cases. You can use the
+`Result` constructor to construct an [`Ok`](#ok), but it will read better to 
+just use [`Ok`](#ok).
+
+```javascript
+import Result from 'crocks/Result'
+
+import and from 'crocks/logic/and'
+import ifElse from 'crocks/logic/ifElse'
+import curry from 'crocks/core/curry'
+import tryCatch from 'crocks/Result/tryCatch'
+import compose from 'crocks/core/compose'
+import chain from 'crocks/pointfree/chain'
+import isDefined from 'crocks/predicates/isDefined'
+import constant from 'crocks/combinators/constant'
+import bimap from 'crocks/pointfree/bimap'
+import identity from 'crocks/combinators/identity'
+
+const { Err, Ok } = Result
+
+// gte :: Number -> Number -> Boolean
+const gte =
+  y => x => x >= y
+
+// lte :: Number -> Number -> Boolean
+const lte =
+  y => x => x <= y
+
+// between :: (Number, Number) -> Boolean
+const between = (x, y) => and(gte(x), lte(y))
```

Comment:
wanna avoid using single line functions.  These should include the arguments on the first line and the body on the next line indented to keep is :corn:sistant with the rest of the functions. So this would be something like:
```js
const between = (x, y) =>
  and(gte(x), lte(y))
```